src/schema/v2/me/myCollectionInfo.ts

```diff
@@ -33,6 +38,42 @@ export const myCollectionInfoFields = {
     type: new GraphQLNonNull(GraphQLInt),
     resolve: ({ artists_count }) => artists_count,
   },
+  collectedArtistsConnection: {
+    description: "A connection of artists in the users' collection",
+    type: artistConnection.connectionType,
+    args: pageable({
+      page: { type: GraphQLInt },
+      size: { type: GraphQLInt },
+    }),
+    resolve: (_root, args, context) => {
+      const { collectionArtistsLoader } = context
+
+      if (!collectionArtistsLoader) return
+
+      const gravityArgs = Object.assign(
+        { total_count: true },
+        omit(convertConnectionArgsToGravityArgs(args), ["page"])
+      )
```

Comment:
```suggestion
      const gravityArgs = { total_count: true, ...omit(convertConnectionArgsToGravityArgs(args), ["page"]) }
```

And still, the open question of why we need to omit the argument `page` (see [this comment](https://github.com/artsy/metaphysics/pull/4127#discussion_r892621289)) and if it is still possible to paginate with the page argument.